src/layout/index.js

```diff
@@ -5,10 +5,47 @@ import Footer from "./footer"
 import Header from "./header"
 import DebugMediaQueries from "../components/debug-media-queries"
 
+import Card from "./../components/card"
+import Grid from "./../components/grid"
+
 const Layout = ({ children }) => (
   <>
     <Header />
     <main>{children}</main>
+
+    <Grid layout="one_third">
+      <Card
+        heading="In ad eiusmod"
+        button={{
+          label: "Learn more",
+          url: "#"
+        }}
+        theme="theme_1"
+        image="/uploads/placeholder-image.jpg"
+        body="<p>Tempor amet voluptate labore qui Lorem laborum ullamco et.</p>"
+      />
```

Comment:
This is an interesting approach. I didn't like it at first glance, but it might actually be the way to go. I'm going to sit with it.

Reply:
I've sat on the fence about this for awhile and I think I like where you're going with this. The grid component becomes more stylistic, which I don't love, but the styles are complicated enough that it could come in handy. And also, if we don't want it we don't have to use it.

My next question is how far we take the grid? I'd like to see it as simple as possible, accounting for our typical patterns. But it'd also be nice for it to remain flexible. Let's keep it and build on it over time.

That said, I think you should look at two adjustments here:

1. See above for comment on the layout prop.
2. I would also add a `<CardGrid />` component. It would serve to pull the card and grid components together, like you're doing manually above.

Reply:
I'm not sure how I feel about adding a `<CardGrid />` component. On a project recently we had components that were built this way (pulling a couple/few components into one) and ended up removing them and going back to using the individual components. This also reminds me of our some of our recent conversation regarding complexity (e.g. `<LinkList />`), should we keep this on the simpler side until it's something we feel is needed in a specific project?